kythe/extractors/bazel/extract.sh

```diff
@@ -43,6 +43,10 @@
 # Also assumes you have extractors installed as per
 # kythe/extractors/bazel/extractors.bazelrc.
 
+# Print our commands for easier debugging and exit after our first failed
+# command so we avoid silent failures.
+set -ex
+
```

Comment:
I'm pretty sure bazel (and bazelisk) exit with non-zero code if any targets fail. But we want to tolerate some targets failing. Or at least our builds depend on that right now.

Reply:
Good point. I thought we were ok since we use `--keep_going` when extracting, but that does still give a nonzero exit code if any part of the build failed.

Reply:
I updated to turn off e for the bazel invocation

Reply:
done